templates/membership/complete.html.twig

```diff
@@ -14,12 +14,12 @@
 {% endblock %}
 
 {% block javascripts %}
-    {% import 'javascript.js.twig' as js %}
-        <script type="text/javascript">
-            Kernel.onLoad(function() {
-                App.runComplete();
-            });
-        </script>
+{% import 'javascript.js.twig' as js %}
```

Comment:
`{% import 'javascript.js.twig' as js %}` is a one-line block.

so this should be: 
```

{% block javascripts %}
    {% import 'javascript.js.twig' as js %}
    <script type="text/javascript">
        Kernel.onLoad(function() {
            App.runComplete();
        });
    </script>
{% endblock %}
```